report-jtreg/src/main/resources/io/jenkins/plugins/report/jtreg/JenkinsReportJckGlobalConfig/config.jelly

```diff
@@ -94,9 +94,9 @@
                         Track a failed test with the list tool
                         <ul>
                             <li><i>Link label:</i> track</li>
-                            <li><i>Base tool page:</i> list.html?=</li>
-                            <li><i>Spliterator:</i> -</li>
-                            <li><i>Tool arguments:</i> <code>-view=all-tests, -output=html, -fill, -track=%{TESTNAME}  %{JOBNAME}  0, -365</code></li>
+                            <li><i>Base tool page:</i> list.html?generated-part=&amp;custom-part=</li>
```

Comment:
Why the others do not need this? I mean the ones calling comparator?

Reply:
It has a different implementation here, it takes the base page verbatim and puts it into the url, with the comparator links, this url is hardcoded. If you wish, I can do a hardcoded dropdown list instead of this with all the possible tool webpages.

Reply:
Can you elaborate a bit more? I think I do not follow. On comaprator urls eg `http://...com:9090/comp.html?generated-part=&custom-part=--compare-traces+--only-vo...` There is also custom part.

I would like tohave this behavior unifed, and I would most likely like to not hardocde anything. So enforcing to use whole  `diff/list/.../comp.html?generated-part=&custom-part=`  everywhere. Those indeed may be used in some editable drop down comobox, but user must have freedom to redirect it to any custom service. wdyt?

Reply:
I'ms till not follwoing why comparator  should be handled differently? Please unify those two... and add the dropdown box with most common usages (the four services with  generated/custom parts)

Reply:
but since i will be updating the class with the links, it will probably drop the current links again

Reply:
one more reason to  have them "backjuped" as examples O:)

But no, they should not get completly lost. Thy will stop working, but the fields itself shold persist. As I see it, you will be changing just one field, and then way how to combine them together, no?

Reply:
Yes, I will be changing just one field